src/library-authoring/library-team/AddLibraryTeamMember.tsx

```diff
@@ -0,0 +1,61 @@
+import { useIntl } from '@edx/frontend-platform/i18n';
```

Comment:
The library-team UI borrows heavily from the course-team UI. I initially tried to factor out the common parts so we could re-use components. But there were so many subtle differences between them -- 3 roles instead of 2, different text, the "allow public" toggle switch -- that it made the code really hard to read and use. So I ended up just copying over what I needed, though I did decide to use some css classes to keep a consistent appearance.